src/ProjectNeon/Assets/Features/Effects/EffectType.cs

```diff
@@ -29,12 +29,14 @@ public enum EffectType
     OnAttacked = 26,
     CostPrimaryResourceEffect = 27,
     ForNumberOfTurns = 28,
-    FeedOnEffect = 29,
+    FeedOnEffect = 29, //unused effect
```

Comment:
Go ahead and delete this if it's unused. As long as we don't reuse the number, and nothing is broken in game, we are good.

Reply:
@SilasReinagel I'll comment all the line, for making sure that we won't reuse the number